File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.java
class Solution {
    public int solution(int n) {
        int count = 0; // 순서쌍 개수 (약수 개수)

        // 1부터 n까지 반복
        for (int i = 1; i <= n; i++) {
            // i가 n의 약수인지 확인
            if (n % i == 0) {
                count++; // 약수면 카운트 증가
            }
        }

        return count; // 결과 반환
    }
}

File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.py
def solution(n):
    count = 0  # 순서쌍 개수

    # 1부터 n까지 반복
    for i in range(1, n + 1):
        if n % i == 0:  # 약수인지 확인
            count += 1  # 개수 증가

    return count  # 결과 반환